ui/package.json

```diff
@@ -42,7 +42,11 @@
     }
   ],
   "dependencies": {
-    "jasmine-core": "^2.5.2"
+    "fsevents": "^2.1.2",
+    "gyp": "^0.5.0",
```

Comment:
why did the package.json changed for python tests?

Reply:
these have come during the setup of the frontend. these packages were somewhat required to complete the setup otherwise a few errors were flashing. any suggestions are most welcome.